src/styles/blocks/navigation.scss

```diff
@@ -0,0 +1,45 @@
+.navigation {
+  &__list {
+    margin: 0;
+    padding: 0;
+    display: flex;
+    list-style-type: none;
+    align-items: end;
```

Comment:
The value 'end' is not a valid value for 'align-items'. Did you mean 'flex-end'? Using 'end' will not have the desired effect and may cause cross-browser issues.